source/scss/_library/_objects.sections.scss

```diff
@@ -38,3 +38,184 @@
     }
   }
 }
+
+
+/**
+ * Block group
+ */
+.c-block-group {
+  padding: $space-double 0 $space;
+
+  @include media('>medium') {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: $space-double 0;
+  }
+
+
+  // Two columns with ad unit
+  .c-block-group__col1 {
+    margin-bottom: $space-triple;
+    @include media('>medium') {
+      padding-right: $gutter + px;
+      width: calc((100% - 300px) - #{$gutter}px);
+    }
+  }
+  .c-block-group__col2 {
+    @include media('>medium') {
+      width: 300px;
+      top: 0;
+      position: sticky;
+    }
+  }
+
+  // With ad
+  &--with-ad {
+    @include media('<=medium') {
+      display: flex;
+      flex-wrap: wrap;
+      flex-flow: column-reverse;
+
+      .c-block-group__col2 {
+        margin-bottom: $space-triple;
+      }
+    }
+  }
+
+  // Single column
+  &--single-col {
+    display: block;
+
+    .c-block-group__col1 {
+      width: 100%;
+      padding: 0;
+
+      > .c-block {
+        margin-bottom: $space-triple;
+      }
+    }
+  }
+}
+
+.c-home {
+  &__sections {
+    padding-top: $space-triple;
+
+    @include media('>medium') {
+      padding-top: $space-quad;
+    }
+  }
+}
+
+/**
+ * Featured block groups
+ */
+.c-featured-blocks {
+  border: 3px solid $c-black;
+  position: relative;
+  margin-right: -$space;
+  margin-left: -$space;
```

Comment:
see above about margin-busting on concrete elements

Reply:
Yeah, good call. Probably more of these coming down the line.